Spigot-Server-Patches/0728-Add-corner-getter-to-WorldBorder.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TwoLeggedCat <80929284+TwoLeggedCat@users.noreply.github.com>
+Date: Wed, 12 May 2021 22:35:57 -0500
+Subject: [PATCH] Add corner getter to WorldBorder
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftWorldBorder.java b/src/main/java/org/bukkit/craftbukkit/CraftWorldBorder.java
+index 31f197ded3335cf6615a7e7d4e502323daad13e4..c36c9a415ec28056e4b6e2d3be59307e05386c3a 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftWorldBorder.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftWorldBorder.java
+@@ -71,6 +71,17 @@ public class CraftWorldBorder implements WorldBorder {
+         this.setCenter(location.getX(), location.getZ());
+     }
+ 
++    //Paper start
++    @Override
++    public @org.jetbrains.annotations.NotNull Location getCornerLocation(@org.jetbrains.annotations.NotNull Corner corner) {
++        org.apache.commons.lang.Validate.notNull(corner, "Corner cannot be null");
++        double halfSize = this.getSize() / 2;
```

Comment:
Add a new method called `#getRadius` or something for this. Can be private if you so please.

Reply:
Don't WorldBorders always have to be rectangles? I'm not really sure what to call it other than `radius`, but that seems strange given it's not a circle

Reply:
Went with "getHalfLength()"